packages/t/tbb/fetch.lua

```diff
@@ -1,11 +1,11 @@
 import("lib.detect.find_path")
 import("lib.detect.find_library")
 
-function _find_package_on_windows(package, opt)
-    local rdir = (package:is_arch("x64") and "intel64" or "ia32")
+function _find_package(package, opt)
+    local rdir = (package:is_arch("x64") and "x86_64" or "ia32")
```

Comment:
I think you misunderstood, @waruqi meant checking if the arch is x86_64.
The correct line would be
```lua
local rdir = (package:is_arch("x64", "x86_64") and "intel64" or "ia32")
```

Reply:
thanks